tests/end_to_end/test_vulnerability_detector/test_vulnerability_detector.py

```diff
@@ -89,7 +89,7 @@
     r"Lost connection with manager. Setting lock",
     r"Unable to connect to .*Connection refused",
     r"Unable to connect to .*No connection could be made because the target machine actively refused it",
-    r"Waiting for server reply \(not started\). Tried: .*Ensure that the manager version is 'v4.8.0' or higher",
+    r"Waiting for server reply \(not started\). Tried: .*Ensure that the manager version is .*",
```

Comment:
One solution might be to replace this with the `get_version()` output. Is there a reason why this wasn't considered?

Reply:
No, this is a backport from 4.10.0, it is the same fix